### PR TITLE
clarify multiple tokens case for request authentication API

### PR DESCRIPTION
**Pull request by yangminzhu**
Clarify the handling of multiple tokens and be consistent with the https://istio.io/latest/docs/concepts/security/#request-authentication

### CODE REVIEW

File: security/v1beta1/jwt.proto [hunk 1]
```diff
@@ -119,6 +119,9 @@ message JWTRule {
   //   - name: x-jwt-assertion
   //     prefix: "Bearer "
   // ```
+  //
+  // Note: Requests with more than one tokens (at different locations) are not supported, the output principal of
```
Comment (ericvn) [inline security/v1beta1/jwt.proto:123]: Maybe
```suggestion
  // Note: Requests with multiple tokens (at different locations) are not supported, the output principal of
```
or maybe `more than one token` (removing the s).

Same comment in the other locations.

Comment (yangminzhu) [reply security/v1beta1/jwt.proto]: done

**Review by ericvn — APPROVED** [review]

File: security/v1beta1/jwt.proto [hunk 2]
```diff
@@ -119,6 +119,9 @@ message JWTRule {
   //   - name: x-jwt-assertion
   //     prefix: "Bearer "
   // ```
+  //
+  // Note: Requests with multiple tokens (at different locations) are not supported, the output principal of
+  // such requests is undefined.
```
Comment (hzxuzhonghu) [inline security/v1beta1/jwt.proto:124]: The comment is good, it is confusing for users the fields below are repeated.

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**